samples/goroutines3.go

```diff
@@ -0,0 +1,41 @@
+package main
```

Comment:
This example doesn't seem to work in the playgrounds (timeout).
I also don't quite understand the point of it - could you please explain?

Reply:
I didn't refer to this one on a slide. So it isn't there on goplay.space. I only wanted to run this on my machine to show how quickly goroutines are spawned and run (to illustrate the 'very lightweight thread' point).